package.json

```
@@ -8,6 +8,8 @@
         "build": "webpack --mode production",
         "lint": "stylelint ./src/**/*.scss && eslint \"src/**/*.{ts,tsx}\"",
         "lint-fix": "stylelint --fix ./src/**/*.scss && eslint --fix \"src/**/*.{ts,tsx}\"",
+        "stylelint": "npx stylelint ./**/*.scss",
+        "stylelint:fix": "npx stylelint ./**/*.scss --fix",
```

бы фиксы(-fix/:fix) заунифицировать
```suggestion
        "stylelint-fix": "npx stylelint ./**/*.scss --fix",
```
или наоборот
```
        "lint:fix": "stylelint --fix ./src/**/*.scss && eslint --fix \"src/**/*.{ts,tsx}\"",
```
Но я за первый вариант больше, потому что когда пишешь команду, то дефис попривычнее в имени команды

---

Я писал с оглядкой на [это](https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/npm-run-all.md)
Но наверное он и с дефисами работает)